_config.yml

```diff
@@ -69,8 +69,8 @@ exclude_from_localizations: ["assets", "admin"]
 header_links: 
   - url: /subscribe
     text: Join
-  - url: /events
-    text: Events       
+  - url: /circuit-breakers
```

Comment:
more important than /events and the /events are still visible in footer/mobile menue